### PR TITLE
Bump to v1.2.0

### PR DESCRIPTION
**Pull request by tiraboschi**
Bump to v1.2.0
Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

**Release note**:
```release-note
Bump to v1.2.0
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
All tests passed

File: version/version.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "1.1.0"
+	Version = "1.2.0"
```
Comment (nunnatsa) [inline version/version.go:4]: This fails some tests in pkg/controller/hyperconverged/hyperconverged_controller_test.go

To fix the tests, it is possible to replace https://github.com/tiraboschi/hyperconverged-cluster-operator/blob/b25a7dfb9dc399c369b7a9dd1e78be0df546916d/pkg/controller/hyperconverged/hyperconverged_controller_test.go#L1963 with this code:
``` golang
BeforeEach(func() {
	os.Setenv(util.HcoKvIoVersionName, version.Version)
})
```

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Pull request merged**